paperqa/agents/models.py

```diff
@@ -354,6 +346,7 @@ def strip_answer(
     async def get_summary(self, llm_model="gpt-4-turbo") -> str:
         sys_prompt = (
             "Revise the answer to a question to be a concise SMS message. "
+            # Force two lines to keep ruff and black happy
```

Comment:
Why do you have this?

Reply:
Black thinks the line length is too long, so breaks it up into two strings on separate lines. 

Ruff thinks the line lengths are too short, so puts the two strings on one line. Then Ruff complains that we're doing string concatenation in one line. 

I think there's incompatible configurations.

Reply:
We don't have `ruff-format` here, so I am not sure why Ruff would be reshaping strings. Lemme check this out and play with it

Reply:
Deleting this line everything seems fine (?)

I think you can just delete this comment and everything passes

Reply:
Seems the ruff VSCode extension was interfering. Resolved!